src/components/DropDownButton/index.tsx

```diff
@@ -84,17 +85,19 @@ const Toggle = ({
 	handler,
 	label,
 	joined,
+	compact,
 	...props
 }: InternalDropDownButtonProps & {
 	open: boolean;
 	handler: React.MouseEventHandler<HTMLElement>;
 }) => {
+	const shouldCompact = useBreakpoint(compact || [false]);
 	if (joined) {
-		if (label || props.icon) {
+		if (label || shouldCompact) {
```

Comment:
I'm not sure I get this change :thinking:
I would expect this to still be `label || props.icon`.
Can I get some context?

Reply:
@thgreasi this change had been made when the compact property was added to the filters. Now since the compact functionality is managed directly by the buttons we can remove that check as it was originally and manage it with the compact property